src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,41 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "When did World Wide Web Consortium first drafted standards for developing accessible websites?",
+    Answer: "1999",
+    Distractor1: "2001",
+    Distractor2: "1990",
+    Distractor3:
+      "1989",
+    Explanation:
+      "The World Wide Web Consortium first drafted standards for developing accessible websites back in 1999.",
+    Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/"
+  },
+  {
+    Question:
+      "What legal requirement has brought web accessibility to the forefront in recent years?",
+    Answer:"The Americans with Disabilities Act",
+    
```

Comment:
fix formatting errors 

```suggestion
    Answer: "The Americans with Disabilities Act",
```